ipatests/test_integration/test_edns.py

```diff
@@ -255,3 +255,19 @@ def install(cls, mh):
             "--dot-forwarder", "1.1.1.1#cloudflare-dns.com"
         ]
         tasks.install_master(cls.master, extra_args=args)
+
+    def test_check_dot_forwarder_added_in_ipa_conf(self):
+        """
+        This test checks that forwarders is listed in
+        dnsserver-show command and also the dot forwarder is
+        added to unbound and included in
+        /etc/unbound/conf.d/zzz-ipa.conf
+        """
+        msg = 'Forwarders: 127.0.0.55'
+        cmd1 = self.master.run_command(
+            ["ipa", "dnsserver-show", self.master.hostname]
+        )
+        assert msg in cmd1.stdout_text
+        with open(paths.UNBOUND_CONF, 'r') as tmpfile:
```

Comment:
This command is executed on the controller. I assume you want to read the file on the master instead?